schema/library_schema.json

```diff
@@ -0,0 +1,11124 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "$id": "http://example.com/example.json",
+    "description": "Placeholder",
+    "type": "object",
+    "default": {},
+    "required": [],
+    "properties": {
+        "SupplyAirTempReset": {
```

Comment:
The schema should be generic for all library items, not specific to each item. It should define what should be in each verification. This file should be pretty short and be similar to the ones I shared as examples. It should basically describe that the expected properties should be the following ones and what type should they be.

- "library_item_id"
- "description_brief"
- "description_detail"
- "description_index"
- "description_datapoints"
- "description_assertions"
- "description_verification_type"
- "assertions_type"

We also need need code that we can use to validate library files against the schema.

Reply:
I see. I will do it.